src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,46 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: {
+    user: {
+      id: number;
+      name: string;
+      username: string;
+      email: string;
+    } | null;
+    id: number;
+    title: string,
+    completed: boolean;
+    userId: number;
```

Comment:
The 'userId' property is not used in your component. If you are not utilizing it, you should remove it to keep the code clean and readable.